dockerfiles/examples/local/README.md

```diff
@@ -11,10 +11,31 @@ NB: The output of installing this chart, will contain a generated example with t
 
 Examples of use:
 
-*helm install localfile hpcc-localfile/*
-*helm install localfile hpcc-localfile/ --set storage.common.hostpath /home/me/hpcc-data*
+#### Docker desktop (using wsl2):
+
+  helm install localfile examples/local/hpcc-localfile --set common.hostpath=/run/desktop/mnt/host/c/wsldata/hpcc
```

Comment:
NB: "wsldata/hpcc" was my example use case.
I think this README should use same root hpcc dir. in all examples, e.g. "hpccdata", changing this to 
```
helm install localfile examples/local/hpcc-localfile --set common.hostpath=/run/desktop/mnt/host/c/hpccdata
```